### DISCUSSION TITLE
寻求新的维护者

### DISCUSSION THREAD

**Discussion by tanukihee** [whutug/whu-thesis#315] [General]
由于 whutug 的成员都已经/即将毕业，后续如果新的需求，可能不会有较快的反馈。因此寻找仍在学校中，对 LaTeX 比较熟悉或者感兴趣维护代码的同学，成为新的维护者。

现在也基本没什么开发方向了，可能中期计划是重写 README，远期计划是与校方联系，得到校方格式上的意见与指导，再远一点的就是学 fduthesis 用 xtemplate 重构封面（逃）

欢迎有兴趣的同学在底下留言

**Comment by imfing** [comment]
我可以负责维护 README（手动狗头

**Comment by sikouhjw** [comment]
我可以负责贡献 star 数（滑稽

**Comment by fpGHwd** [comment]
我也要毕业了……否则很想接（不

**Comment by earthmoon** [comment]
我很想加入，但我不会，正在学，也不知道怎么才能学好。

**Comment by sikouhjw** [comment]
> 我很想加入，但我不会，正在学，也不知道怎么才能学好。

先把 lshort-zh-cn 读完，然后去看 LaTeX3

**Comment by earthmoon** [comment]
> > 我很想加入，但我不会，正在学，也不知道怎么才能学好。
> 
> 先把 lshort-zh-cn 读完，然后去看 LaTeX3

LaTeX3 有链接吗？我只找到lshort-zh-cn

**Comment by sikouhjw** [comment]
> > > 我很想加入，但我不会，正在学，也不知道怎么才能学好。
> > 
> > 
> > 先把 lshort-zh-cn 读完，然后去看 LaTeX3
> 
> LaTeX3 有链接吗？我只找到lshort-zh-cn

我是说学 LaTeX3 的文档，我也忘了名字了

**Comment by earthmoon** [comment]
> > > > 我很想加入，但我不会，正在学，也不知道怎么才能学好。
> > > 
> > > 
> > > 先把 lshort-zh-cn 读完，然后去看 LaTeX3
> > 
> > 
> > LaTeX3 有链接吗？我只找到lshort-zh-cn
> 
> 我是说学 LaTeX3 的文档，我也忘了名字了

哪天想起了，记得告诉我啊

**Comment by tanukihee** [comment]
`texdoc interface3`

但在学习 l3 的过程中，你会不可避免地需要了解 plain tex 相关的东西…

**Comment by earthmoon** [comment]
> `texdoc interface3`
> 
> 但在学习 l3 的过程中，你会不可避免地需要了解 plain tex 相关的东西…

好抽象

**Comment by redhat123456** [comment]
我可以吗？

很想加入你们的论文排版的设计，以及参与维护工作

🌈🌈

**Comment by tanukihee** [comment]
> 我可以吗？
> 
> 很想加入你们的论文排版的设计，以及参与维护工作
> 
> 🌈🌈

由于需要跟进校内的通知，以及与校内老师沟通交流，所以希望最好还是武大的同学们来接手

**Comment by tanukihee** [comment]
当然也欢迎各位同学们提交 pr～

**Comment by redhat123456** [comment]
好的   🌈🌈

**Comment by fyqqyf** [comment]
我是一名大四学生正准备写毕业论文，之前业余也做过beamer、LaTeX模板。本来想参与维护的工作，但实际clone了该项目以后，发现自己以前做的东西还是太naive了……（所以有大佬维护**2022WHU-thesis**吗🥺 ）

**Comment by sikouhjw** [comment]
> 我是一名大四学生正准备写毕业论文，之前业余也做过beamer、LaTeX模板。本来想参与维护的工作，但实际clone了该项目以后，发现自己以前做的东西还是太naive了……（所以有大佬维护**2022WHU-thesis**吗🥺 ）

不要紧，可以学的

**Comment by fyqqyf** [comment]
> > 我是四学生正准备写毕业论文，之前的空闲时间也可以再次开灯、LaTeX 模板。本来想参加维护的工作，但实际克隆了该项目以后，发现自己以前做的东西还是太天真了…… （所以有大佬维护**2022WHU-thesis**吗？🥺 ）
>
> 不要紧，可以学的

其实学习过lshort-zh-cn，平时正常使用LaTeX也比较熟悉，但对于这种大型(?)项目的维护并没有经验把握。🤣 后面或许可以根据**学校最新的格式要求**调整模板和提交pr，做一些力所能及的贡献。

**Comment by imfing** [comment]
@fyqqyf 可以可以，欢迎Contribute～

**Comment by xkwxdyy** [comment]
目前是武大数统在读研一，学过LaTeX3基本知识，会使用常见的模块，虽然能力可能不足，但也愿意学习。目前是CCNUthesis（非官方）的维护者

**Comment by tanukihee** [comment]
> 目前是武大数统在读研一，学过LaTeX3基本知识，会使用常见的模块，虽然能力可能不足，但也愿意学习。目前是CCNUthesis（非官方）的维护者

终于抓到壮丁了，简单列一列 TODO List

+ 给 main.tex 多补点注释，不要只写博士的，也写一写硕士本科应该怎么写（被评论「读文档要读到 2.3 节去，人早没了」）
+ 把各种常量配置给拿出来单独放进一个文件里，不要写死在 sty 里
+ 收到了外院的英语和德语的 word 模板，不过想要用 LaTeX 实现的话可能有点困难（
+ 修 issue（我太懒了
+ 用 xtemplate 重构封面
+ ……

**Comment by imfing** [comment]
> 终于抓到壮丁了，简单列一列 TODO List
> 
> * 给 main.tex 多补点注释，不要只写博士的，也写一写硕士本科应该怎么写（被评论「读文档要读到 2.3 节去，人早没了」）
> * 把各种常量配置给拿出来单独放进一个文件里，不要写死在 sty 里
> * 收到了外院的英语和德语的 word 模板，不过想要用 LaTeX 实现的话可能有点困难（
> * 修 issue（我太懒了
> * 用 xtemplate 重构封面
> * ……

@tanukihee 建议给 cls 文件多写点注释...

**Comment by wangzhankun** [comment]
这个要不成立一个QQ群吧，好相互联系,也便于后续更新支持，大家相互讨论起来可能更方便一些。 @tanukihee @xkwxdyy

**Comment by tanukihee** [comment]
> 这个要不成立一个QQ群吧，好相互联系,也便于后续更新支持，大家相互讨论起来可能更方便一些。 @tanukihee @xkwxdyy 

有一个群，681965476

**Comment by sikouhjw** [comment]
> > 这个要不成立一个QQ群吧，好相互联系,也便于后续更新支持，大家相互讨论起来可能更方便一些。 @tanukihee @xkwxdyy
> 
> 有一个群，681965476

才四个人，这让我怎么水（迫真）

**Comment by xkwxdyy** [comment]
> 才四个人，这让我怎么水

@sikouhjw 毕竟，我还没重写完，剩下一个硕博的参考文献鸽了很久（doge）

而且还有一堆 issue 没搞（逃

**Comment by taoxiom** [comment]
> > 才四个人，这让我怎么水
> 
> @sikouhjw 毕竟，我还没重写完，剩下一个硕博的参考文献鸽了很久（doge）
> 
> 而且还有一堆 issue 没搞（逃

学长这么强，希望明年我写毕业论文的使用能用上rebuild（😎

**Comment by xkwxdyy** [comment]
> 学长这么强，希望明年我写毕业论文的使用能用上rebuild（😎

逮住又一个维护者（好诶

**Comment by xiahuatian** [comment]
我可以，本人武汉某高校学生，非武大，有基础，对该项目非常感兴趣。不知可不可以？

**Comment by xkwxdyy** [comment]
> 我可以，本人武汉某高校学生，非武大，有基础，对该项目非常感兴趣。不知可不可以？

你开发过什么 LaTeX 相关的项目吗

**Comment by SwitWu** [comment]
我和鱼羊也即将毕业了，有想加入维护团队的同学可以看看[开发者须知](https://github.com/whutug/whu-thesis/blob/main/To-Developers.md)，欢迎有兴趣、有能力的同学加入！

**Comment by cqc717** [comment]
最近在写毕业论文，发现了一些需要设置的地方，可否加入呢？

**Comment by SwitWu** [comment]
> 最近在写毕业论文，发现了一些需要设置的地方，可否加入呢？

你好，可以简单介绍一下你对 LaTeX 语言的了解和掌握程度吗？也建议你说一下模版哪些地方需要进行设置。谢谢！

**Comment by xkwxdyy** [comment]
> 最近在写毕业论文，发现了一些需要设置的地方，可否加入呢？

你有兴趣我们当然很欢迎，但是有几点想说明。（下面的问题也适用于后面有意向维护模板的同学）

1. 你是本 or 硕 or 博？毕业后还在武大吗？
这个问题主要想知道你能大概维护多久，因为从接手，到开始维护，这是有时间和学习成本的，我们倾向于一些低年级同学来维护，如果你只是觉得模板本身有一点不足，欢迎提 issue 和 pr，成为维护者听上去很牛，但其实你要做的事可能比你想的要多很多。
2. 你对 LaTeX 掌握多少？较为系统地学过 LaTeX3 吗？
因为目前绝大部分源码都是 LaTeX3 写的，你如果不太会的话，维护起来也会比较困难
3. 是否开发过一些 LaTeX 模板？了解cls、sty 等的基础内容
4. 是否会 git？如果会，大概掌握到什么程度？
5. 你平时是否有空余时间投入到 LaTeX 中？
直白地说，其实开发模板是一件费力不讨好的事，至少目前而言，只是为爱发电，你确定有这个觉悟吗？

可能这些问题比较劝退，但是还是希望有意向成为维护者的同学能仔细思考并回复上述问题。

**Comment by xiahuatian** [comment]
我本人目前是武汉某高校研究生，不是武大的，latex2e基本OK，学过LaTeX3基本知识，会常见语法和模块，cls、sty等都会，模板套用修改过很多，擅长gpt，可以抽一部分时间投入维护。



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2025年1月2日(周四) 晚上11:58
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [whutug/whu-thesis] 寻求新的维护者 (#60)




  
最近在写毕业论文，发现了一些需要设置的地方，可否加入呢？
  
你有兴趣我们当然很欢迎，但是有几点想说明。（下面的问题也适用于后面有意向维护模板的同学）
  
你是本 or 硕 or 博？毕业后还在武大吗？
 这个问题主要想知道你能大概维护多久，因为从接手，到开始维护，这是有时间和学习成本的，我们倾向于一些低年级同学来维护，如果你只是觉得模板本身有一点不足，欢迎提 issue 和 pr，成为维护者听上去很牛，但其实你要做的事可能比你想的要多很多。
 
你对 LaTeX 掌握多少？较为系统地学过 LaTeX3 吗？
 因为目前绝大部分源码都是 LaTeX3 写的，你如果不太会的话，维护起来也会比较困难
 
是否开发过一些 LaTeX 模板？了解cls、sty 等的基础内容
 
是否会 git？如果会，大概掌握到什么程度？
 
你平时是否有空余时间投入到 LaTeX 中？
 直白地说，其实开发模板是一件费力不讨好的事，至少目前而言，只是为爱发电，你确定有这个觉悟吗？
  
可能这些问题比较劝退，但是还是希望有意向成为维护者的同学能仔细思考并回复上述问题。

**Comment by imfing** [comment]
@xiahuatian @cqc717 

欢迎直接提交Pull Request
